applications/hotornot/src/hon_util.erl

```diff
@@ -8,9 +8,10 @@
 %%%-------------------------------------------------------------------
 -module(hon_util).
 
--export([candidate_rates/1, candidate_rates/2
-         ,matching_rates/2, matching_rates/4
+-export([candidate_rates/3, candidate_rates/4
```

Comment:
This breaks `hotornot_maintenance`. Please update that module's functions that rely on these exports.